coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -5196,3 +5196,22 @@ def trace(context, node):
     diagonal = mb.gather_nd(x=x, indices=indices)
     trace = mb.reduce_sum(x=diagonal, name=node.name)
     context.add(trace)
+
+@register_torch_op
+def roll(context, node):
+    inputs = _get_inputs(context, node, expected=3)
+    x = inputs[0]
+    shift = inputs[1]
+    if inputs[2].val:
+        raise NotImplementedError(f"dims is not supported. {inputs[2]} {inputs[2].val}")
```

Comment:
I don't think printing out `{inputs[2]}` and `{inputs[2].val}` is useful. Let's add more context to the error message.

How about just `'"dims" parameter of "roll" op is not supported'.`